pom.xml

```diff
@@ -1205,6 +1205,13 @@
                     --add-opens java.management/sun.management=ALL-UNNAMED
                     --add-exports java.xml/jdk.xml.internal=ALL-UNNAMED
                     --illegal-access=warn
+                    <!--Allow additional access as a workaround of https://openjdk.java.net/jeps/403  for JDK17 -->
```

Comment:
Put here please also a link to your investigation document or/and to this PR so it's easy to find why it was done this way.

Reply:
@kwart I'm going to add the document to the repository since the community doesn't have access to my google dock and Jira, if you don't mind

Reply:
I think the link to this PR is fine, but also I like the idea of listing modules and classes that require modules to be open:
```
open java.base/java.lang:
  - OurClass1
  - OurClass2
```